src/devices/oil_watchman_advanced.c

```diff
@@ -0,0 +1,96 @@
+/** @file
+    Watchman Sonic Advanced/Plus oil tank level monitor.
+    
+    Copyright (C) 2023 Gareth Potter
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+/**
+Watchman Sonic Advanced/Plus oil tank level monitor.
+
```

Comment:
Can you add a bitstream description here?

Something like the following:

https://github.com/merbanan/rtl_433/commit/cd90897b7f7359423058f392c9e786d96ca4c3b8#diff-afbc9a3bf0ede0786b529b7817b9e59c6f8033ab60f711cdb4ace505c3c03ca6R35